Comments/ClientApp/src/components/CommentPanel/CommentPanel.scss

```diff
@@ -1,31 +1,14 @@
 .CommentPanel {
   background: #eee;
-  position: fixed;
-  right: 0;
-  top: 0;
-  width: 30vw;
-  z-index: 1;
-  height: 100vh;
   padding: rem($spacing-medium);
   &.inactive {
-    transform: translateX(30vw);
-    transition: transform 0.2s ease-out;
+    transform: translateX(240px) !important;
```

Comment:
pixels? why the change in unit?

Reply:
I thought maybe the comment panel would be of a fixed width rather than proportional - I'm sure this will change regardless as decisions are made!